signalfx-java/src/main/java/com/signalfx/signalflow/Channel.java

```diff
@@ -0,0 +1,97 @@
+package com.signalfx.signalflow;
+
+import java.io.Closeable;
+import java.util.Iterator;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Copyright (C) 2016 SignalFx, Inc. All rights reserved.
```

Comment:
Copyright should be a distinct comment at the top of the file (before `package`).